MeshLib/Elements/FaceRule.cpp

```diff
@@ -15,19 +15,29 @@
 
 namespace MeshLib
 {
-
 bool FaceRule::testElementNodeOrder(const Element* e)
 {
-    MathLib::Vector3 up_vec (0,0,1);
-    return (MathLib::scalarProduct(getSurfaceNormal(e), up_vec) < 0) ? true : false;
+    MathLib::Vector3 const z_unit_vector(0, 0, 1);
+    return MathLib::scalarProduct(getSurfaceNormal(e), z_unit_vector) < 0;
```

Comment:
The MeshLib.a is by 80 bytes smaller (so presumably faster). Didn't see that because the change was automatic with clang-tidy. And if I use clang-tidy I switch off my brain...

Actually this whole thing can be replaced with `return true` without any effects on the tests. Furthermore the promise of the `testElementOrder` is not fulfilled for any element with more than three nodes.